WooCommerce/Classes/Tools/UserAgent.swift

```diff
@@ -42,9 +47,9 @@ private extension UserAgent {
         ///
         static let woocommerceIdentifier = "wc-ios"
 
-        /// Load User Agent JS Script
+        /// User Agent Key
         ///
-        static let loadUserAgentScript = "navigator.userAgent"
+        static let userAgentKey = "_userAgent"
```

Comment:
just curious, how does this key come from?

Reply:
It's a property of `WKWebView`, which is not exposed, and it's accessible only calling `.value()` that returns the value for the property identified by a given key. In this case, the key is `_userAgent`. [Here](https://github.com/WebKit/webkit/blob/39a299616172a4d4fe1f7aaf573b41020a1d7358/Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm#L4272) there are all the properties. I also noticed that also on [Wordpress-iOS](https://github.com/wordpress-mobile/WordPress-iOS/blob/7f9dd872ef8c42af866c998840a3be2b6873661a/WordPress/Classes/Extensions/WKWebView%2BUserAgent.swift) we are using the same solution (it seems the only way for accessing to the user agent on WKWebView).